libs/components/src/trip-form-create/steps/LocationStep.tsx

```diff
@@ -0,0 +1,60 @@
+import { Field } from 'formik'
+import React, { useEffect } from 'react'
+import { Column, Heading, Input, Row } from '@packup/components'
+
+export default function LocationStep(props: any) {
+  const {
+    formField: { startingPoint, lat, lng, name },
+    formValues,
+    setFieldTouched,
+    setFieldValue,
+  } = props;
+
+  useEffect(() => {
+    if (formValues[startingPoint.name]) {
+      setFieldValue(startingPoint.name, formValues[startingPoint.name])
+      setFieldTouched(startingPoint.name)
+    }
+
+    // Set title to starting point if it's not already set
+    if (formValues[name.name].length === 0) {
+      setFieldValue(name.name, formValues[startingPoint.name])
```

Comment:
love this idea! maybe prepend with "Trip to" like `Trip to ${formValues[startingPoint.name]}` so it reads a little better?